temba/locations/management/commands/import_geojson.py

```diff
@@ -15,44 +16,55 @@ class Command(BaseCommand): # pragma: no cover
     args = '<file1.zip | 49915admin1.json.. >'
     help = 'Import our geojson zip file format, updating all our OSM data accordingly.'
 
-    def import_file(self, filename, file):
-        admin_json = geojson.loads(file.read())
+    def get_country_id(self, props):
+        if props.get('admin_level') is COUNTRY_LEVEL:
+            return props.get('osm_id')
+        return props.get('is_in_country')
+
+    def import_file(self, file_obj, country_id):
+        admin_json = geojson.loads(file_obj.read())
 
         # we keep track of all the osm ids we've seen because we remove all admin levels at this level
         # which weren't seen. (they have been removed)
         seen_osm_ids = []
 
-        # parse our filename.. they are in the format: 192787admin2_simplified.json
-        match = regex.match(r'(\w\d+)admin(\d)(_simplified)?\.json$', filename, regex.V0)
-        if not match:
-            print "Skipping '%s', doesn't match file pattern." % filename
```

Comment:
This breaks compatibility with the current extracts. If you want to import geojson in a different format then you need either a new command or to support both formats.